README.md

```diff
@@ -283,6 +304,17 @@ spec:
         requests:
           cpu: "2.0"
           memory: "4Gi"
+      # If set to true, runner pod container only 1 container that's expected to be able to run docker, too.
+      # image summerwind/actions-runner-dind or custom one should be used with true -value
+      dockerdWithRunner: false
```

Comment:
Can we name this `dockerWithinRunnerContainer` instead and also rename `dockerdResources` to `dockerdContainerResources` so that they are clearer?

Reply:
Hardest part of development is usually to figure out good names :)